src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts

```diff
@@ -361,26 +361,9 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
 			pinned: !options.preview
 		};
 		let trusted = uri.scheme === Schemas.untitled;
-		let model = new NotebookInputModel(uri, undefined, trusted, undefined, undefined, undefined, options.connectionId);
```

Comment:
You've lost connectionId being passed through. This will mean we regress connection passing from the dashboard / object explorer scenarios. See my other notes on how to fix - basically you should pass through to the NotebookInput and just let it set these on the model in a consistent way. Thanks!